packages/labs/analyzer/src/lib/analyze-package.ts

```diff
@@ -4,14 +4,21 @@ import * as path from 'path';
 import {DiagnosticsError} from './errors.js';
 import {Analyzer} from './analyzer.js';
 
+export interface AnalyzerOptions {
+  exclude?: string[];
```

Comment:
Done.

Yeah, globs. Useful for excluding things source like test folders that might otherwise be included in a project's tsconfig.

It might be interesting (later on, not now) to analyze a package.json's `files` or `exports` and filter down the input source to analyze based on that. 

However ^^ highlights an unsolved problem in the analyzer right now (will likely need to be resolved via config) which is that just because we can determine that `src/foo.ts` gets output by tsc to `build/foo.js` doesn't mean that's where the file is ultimately importable from, since additional transforms outside of ts can (and often do) further process that into another location that is ultimately published (i.e. we have `tsc` output to `./development/*.js` and process that further with rollup to `./*.js`). 

This is something we're going to hit pretty quickly when we start testing against real-world packages. It's been on my radar, but went ahead and added https://github.com/lit/lit/issues/3578 to track it.

Reply:
I think there are a few things we can do here that won't cover _every_ case, but will cover a lot of them... I'll comment on the issue.

The biggest thing that clicked for me here is that these augment the tsconfig exclude setting.